shared/src/main/kotlin/com/egm/stellio/shared/util/QueryUtils.kt

```diff
@@ -0,0 +1,47 @@
+package com.egm.stellio.shared.util
+
+import com.egm.stellio.shared.model.QueryParams
+import org.springframework.util.MultiValueMap
+
+fun parseAndCheckParams(
+    pagination: Pair<Int, Int>,
+    queryParams: MultiValueMap<String, String>,
+    contextLink: String
+): QueryParams {
+
+    val ids = queryParams.getFirst(QUERY_PARAM_ID)?.split(",")?.toListOfUri()
+    val type = queryParams.getFirst(QUERY_PARAM_TYPE)?.let {
```

Comment:
check 6.4.3.2 (v1.4.1 or before since multityping has been introduced in v1.5.1): `type` is a "list of entity types to be retrieved". I think entity service only supports one (to be checked) but search service supports a list, so you can't just restrict to one without breaking existing usages in search service.